dao/searchDao.js

```diff
@@ -0,0 +1,14 @@
+const db = require('./db');
+
+exports.getAll = function(filter, cb) {
+    let classFilter = '';
+    if(filter.class) {
+        classFilter = ' AND class = ' + db.escape(filter.class);
+    }
+    let sqlQuery = 'SELECT * FROM tbl_tickets WHERE reserver IS NULL' + classFilter + ';';
+    // filter out reserver by default because we want tickets that have not been taken yet
+    db.query(sqlQuery,
+    function(err, result) {
```

Comment:
You can simply pass `cb` here instead of wrapping it in a lambda that does nothing more than call it.

Reply:
Yeah, I tried that, but it just gave me an error

Reply:
I suspect you were using the wrong syntax, then. This should work:

```javascript
db.query(sqlQuery, cb);
```